akka-http/src/main/scala/akka/http/scaladsl/server/RejectionHandler.scala

```diff
@@ -165,7 +165,11 @@ object RejectionHandler {
       }
       .handle {
         case TooManyRangesRejection(_) ⇒
-          complete((RequestedRangeNotSatisfiable, "Request contains too many ranges."))
+          complete((RequestedRangeNotSatisfiable, "Request contains too many ranges"))
+      }
+      .handle {
+        case CircuitBreakerOpenRejection(_) ⇒
+          complete((EnhanceYourCalm, "Circuit Breaker is open; calls are failing fast"))
```

Comment:
Also `EnhanceYourCalm` somewhat implies that it's the _client_ that's at fault, which need not be the case here (it's  one of the server's own dependencies that broken)